src/llama.cpp

```diff
@@ -4880,6 +4899,18 @@ static void llm_load_hparams(
                     default: model.type = e_model::MODEL_UNKNOWN;
                 }
             } break;
+        case LLM_ARCH_JAIS:
+            {
+                ml.get_key(LLM_KV_ATTENTION_LAYERNORM_EPS, hparams.f_norm_eps);
+                // TODO: become GGUF KV parameter
+                hparams.f_max_alibi_bias = 8.0f;
```

Comment:
There is already a `<arch>.attention.max_alibi_bias` parameter. I know that this was copied from other archs that still hardcode this parameter, but I don't think we should do this for new archs. Instead, it should be added as metadata in `convert-hf-to-gguf.py` with `gguf_writer.add_max_alibi_bias`.

Reply:
Ah perfect
I'll update it shortly